src/main/java/uk/gov/hmcts/darts/arm/client/ArmRpoClient.java

```diff
@@ -115,4 +118,11 @@ RemoveProductionResponse removeProduction(@RequestHeader(AUTHORIZATION) String b
     )
     ExtendedProductionsByMatterResponse getExtendedProductionsByMatter(@RequestHeader(AUTHORIZATION) String bearerToken,
                                                                        @RequestBody String body);
+
+    @GetMapping(value = "${darts.storage.arm-api.rpo-url.download-production-path}")
+    @SuppressWarnings({"PMD.UseObjectForClearerAPI"})
+    feign.Response downloadProduction(@RequestHeader(AUTHORIZATION) String bearerAuth,
```

Comment:
Do we need to also specify a content type e.g. `consumes = APPLICATION_OCTET_STREAM_VALUE`?